default.nix

```diff
@@ -89,28 +89,77 @@ in
     buildInputs = [
       run-unit-test
       build-hp-admin
```

Comment:
I am not sure what is the purpose of this import here. `build-hp-admin` is essentially a shell script that installs npm modules and runs the `build` command. Same functions are encoded in `preConfigure` and `buildPhase` below. Would love to hear an opinion from @samrose or @yegortimoshenko on this.

Reply:
@peeech This was just added for the users of the repo(hAppy team) to build their project using nix.

Reply:
ok, thanks for clarification